src/ralph/admin/autocomplete.py

```diff
@@ -35,7 +35,9 @@ def get(self, request, *args, **kwargs):
         """
         Returns serialized dict as JSON object.
         """
-        can_edit = ralph_site._registry[self.model].has_change_permission(
+        can_edit = not getattr(
```

Comment:
please add also TODO comment that for polymorphic models edit should be enabled, but should redirect to editing concrete (final) object.